mockwebserver/src/main/kotlin/mockwebserver3/StreamHandler.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2018 Square, Inc.
+ * Copyright (C) 2022 Block, Inc.
```

Comment:
Global search and replace? Or just new files in the year of our lord 2023?

Reply:
Block for new things, old things stay put.